recipes/jonga/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jonga" %}
+{% set version = "0.0.4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d258421c66887cb10122d3237e06bc5b14a4809fdcef4654b62dc2e397fb0892" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # Skip windows build since pygraphviz dependency not available
+  # noarch: python
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.3
+    - setuptools
+    - pip
+    - pygraphviz
+    - pytest
```

Comment:
Are pytest and pytest-runner needed for the package, or only for tests?

Reply:
In principle they're only needed for the tests, but the most recent release on PyPi has them listed as install requirements, so they're needed here too, for now.